src/ui/units/dash/containers/Body/Body.tsx

```diff
@@ -858,13 +874,14 @@ class Body extends React.PureComponent<BodyProps> {
             dashkitSettings,
         } = this.props;
 
+        const {getPreparedCopyItemOptions: getOptions, ...context} = this.getContext();
```

Comment:
Let's remove this method from memo https://github.com/datalens-tech/datalens-ui/blob/bf7e5a6be4a303058810be7264b47a262152ea98/src/ui/units/dash/containers/Body/Body.tsx#L705-L706
And use this `fn` https://github.com/datalens-tech/datalens-ui/blob/bf7e5a6be4a303058810be7264b47a262152ea98/src/ui/units/dash/containers/Body/Body.tsx#L694-L695 as class method that would be passed as `getPreparedCopyItemOptions` prop